modes/play/objects/badGuy.js

```diff
@@ -0,0 +1,35 @@
+class BadGuy extends Mob {
+  constructor(game, mode, x, y) {
+    super(game, mode, x, y);
+    this.color = "#ee5";
+    this.brain = new BadGuyBrain(this);
+  }
+}
+
+
+class BadGuyBrain {
+  constructor(meat) {
+    this.meat = meat;
+    this.actionLoop = [
+      'right',
+      'left',
+    ];
+    this.loopPosition = 0;
+    this.state = this.actionLoop[0];
+    this.timer = 0;
+    this.timerLength = 200;
+  }
+
+  get actions() {
+    const jump = Math.random() < 0.01 ? 'jump' : 'just chill'
+    return [this.state, jump];
```

Comment:
Whatever is in this array is what the meat will process. Stupidly, these events are not idempotent, so if the return value is `[ 'left', 'left', 'left' ]`, the mob will move left 3x as fast as if it was just `[ 'left' ]`.

Derp.